sdk/nodejs/tests/runtime/props.spec.ts

```diff
@@ -95,18 +143,111 @@ describe("runtime", () => {
         }));
         it("marshals secrets correctly", asyncTest(async () => {
             runtime._setTestModeEnabled(true);
+
             const inputs: Inputs = {
                 "secret1": secret(1),
                 "secret2": secret(undefined),
             };
+
             // Serialize and then deserialize all the properties, checking that they round-trip as expected.
-            const transfer = gstruct.Struct.fromJavaScript(
+            runtime._setFeatureSupport("secrets", true);
+            let transfer = gstruct.Struct.fromJavaScript(
                 await runtime.serializeProperties("test", inputs));
-            const result = runtime.deserializeProperties(transfer);
+            let result = runtime.deserializeProperties(transfer);
+            assert.ok(runtime.isRpcSecret(result.secret1));
+            assert.ok(runtime.isRpcSecret(result.secret2));
+            assert.strictEqual(runtime.unwrapRpcSecret(result.secret1), 1);
+            assert.strictEqual(runtime.unwrapRpcSecret(result.secret2), null);
+
+            // Serialize and then deserialize all the properties, checking that they round-trip as expected.
+            runtime._setFeatureSupport("secrets", false);
+            transfer = gstruct.Struct.fromJavaScript(
+                await runtime.serializeProperties("test", inputs));
+            result = runtime.deserializeProperties(transfer);
```

Comment:
For completeness, probably should assert `!runtime.isRpcSecret` on these props

Reply:
Sure--that should help verify that `isRpcSecret` works properly.

Reply:
I'll add this as a follow-up as well.